tests/cephfs/cephfs_utilsV1.py

```diff
@@ -574,3 +589,18 @@ def remove_fs(self, client, vol_name, validate=True, **kwargs):
             volname_ls = json.loads(out.read().decode())
             if vol_name in [i["name"] for i in volname_ls]:
                 raise CommandFailed(f"Creation of filesystem: {vol_name} failed")
+
+    def fs_client_authorize(
```

Comment:
This is becoming redundant can we make it only cephfs_utilsV1.py file only.
And port other functions which are required for test case file execution.
So that we can decpricate cephfs_utils.py file one day.
We will port functions so that it will work for both 4.x and 5.x

Reply:
Sure @psathyan , Is there specific format for documentation needed or just explanatory of function

Reply:
Apologies @yogesh-mane for the late response. We recommend following the suggestions at 
https://google.github.io/styleguide/pyguide.html

In short, a few lines on the purpose of the method. Along with the short description on the arguments. This allows others to reuse the method and call it correctly.